src/components/Text/Text.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from 'styled-components/native';
+
+const StyledText = styled.Text`
+  color: ${({ light }) => (light ? '#999' : '#36415a')};
```

Comment:
Imo would be good to use themes. Even a single theme removes the need of having separate variables for colors, sizes etc.